src/Agent.Listener/Configuration/AutoLogonRegistryManager.cs

```diff
@@ -286,7 +286,7 @@ private void UpdateUserSpecificRegistrySettings(CommandSettings command, string
 
         private void UpdateScreenSaverSettings(CommandSettings command, string securityId)
         {
-            if(!command.GetDisableScreenSaver())
+            if(command.GetKeepScreenSaver())
```

Comment:
then, i don't think we need the prompt, we have to make sure screen saver is off during autologon config, isn't it?